tests/e2e/runtime.go

```diff
@@ -319,6 +339,14 @@ func (sc *RuntimeScenario) Run(childEnv *env.Env) error {
 			return testErr
 		}
 		sc.Logger.Info("test passed", "test", testName)
+
+		// Do an invariants check after each test.
```

Comment:
Can you also make this part of the txgen fuzzing test(s) to make sure invariants are being maintained?